DEVELOPMENT.md

```diff
@@ -54,10 +54,32 @@ hatch run test
 ```
 
 ## Run integration tests
+- Pre-requisites:
+  - Developer must have a valid AWS account.
+  - Developer must create a Farm, Queue. This can be done easily via Deadline Cloud's AWS Console quick farm create workflow. Please copy the Farm, Queue resource IDs for the next step. Please copy the Queue's Job Attachment S3 bucket name for the next step.
+
+- Setup Environment Variables
+```bash
+export SERVICE_ACCOUNT_ID=001122334455
+export JOB_ATTACHMENTS_BUCKET="{deadline queue configured Job Attachments S3 bucket.}"
+export JA_TEST_ROOT_PREFIX="/your/path/root"
+export FARM_ID="farm-{uuid}"
```

Comment:
nit: I like the "your <thing>" pattern for the values people should be filling in, just so it is super obvious. Can we do that for all these env vars? "your AWS account ID", "your job attachments bucket name", etc.

Reply:
I like that! let me change it